src/styles/blocks/_calendar.scss

```diff
@@ -0,0 +1,56 @@
+.calendar {
+  width: $calendar-width;
+  padding-left: $calendar-block-padding;
+  padding-top: $calendar-block-padding * 2; 
+  // Doubled padding-top, 
+  // so the top row don't go over top of the page on hover.
+  
+  display: flex;
+  flex-wrap: wrap;
+  gap: $gap;
+
+  &__day {
+    width: $day-block-size;
+    height: $day-block-size;
+    
+    display: flex;
+    justify-content: center;
+    align-content: center;
+    align-items: center;
+
+    font-family: Arial, Helvetica, sans-serif;
+    font-size: $day-block-size * 0.3;
+    
+    border: 1px $border-col solid;
+    background-color: $day-block-col;
+    box-sizing: border-box;
+
+    transition: background-color 0.5s, transform 0.5s;
+
+    &:hover {
+      cursor: pointer;
+      background-color: $hover-day-block-col;
+      transform: translate(0, -20px);
+    }
+  }
+
+  @for $count from 28 through 31 {
+    &--days-count--#{$count} 
+    div:nth-child(n + #{$count + 1}) {
+      display: none;
+    }
```

Comment:
you could move this up. in &__day. maybe it's better for structure BEM

Reply:
you're right! I don`t really know why I use there div -_-*